common/tools/dev-tool/src/commands/run/startBrowserRelay.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license
+
+import { leafCommand, makeCommandInfo } from "../../framework/command";
+
+import { startRelayServer } from "../../util/browserRelayServer";
+
+export const commandInfo = makeCommandInfo(
+  "start-browser-relay",
+  "Start the browser credential relay, used for authenticating browser tests.",
+  {
+    listenHost: {
+      kind: "string",
+      default: "localhost",
+      description: "Host to listen on",
+    },
+    port: {
+      kind: "string",
+      default: "4895",
```

Comment:
should we also let users to provide an ENV variable just like they do for test proxy incase the port is blocked?

Reply:
doesn't have to block this PR, can be incremental

Reply:
We could but we would also have to plumb the variable through using karma.conf etc. Maybe we can revisit if this becomes an issue?

Reply:
Yeah we can definitely revisit.
Only the affected dev can add to their karma.conf, we can deal with that lazily.